code/modules/hydroponics/plants_herb.dm

```diff
@@ -136,6 +136,14 @@ ABSTRACT_TYPE(/datum/plant/herb)
 	/datum/plantmutation/cannabis/white,/datum/plantmutation/cannabis/ultimate)
 	commuts = list(/datum/plant_gene_strain/resistance_drought,/datum/plant_gene_strain/yield/stunted)
 
+	New()
+		. = ..()
+		START_TRACKING_CAT(TR_CAT_CANNABIS_OBJ_ITEMS)
+
+	disposing()
+		. = ..()
```

Comment:
Generally in disposing I'd prefer calling parent to be the last action. Think if disposing as the closing bracket to New being the opening bracket off the object. It doesn't make a change here but it's good practice.

Reply:
Applies to other disposing in this PR too.

Reply:
Done!